worker/transcribee_worker/worker.py

```diff
@@ -51,7 +51,11 @@ def __init__(
         if task_types is not None:
             self.task_types = task_types
         else:
-            self.task_types = [TaskType.DIARIZE, TaskType.ALIGN, TaskType.TRANSCRIBE]
+            self.task_types = [
+                TaskType.IDENTIFY_SPEAKERS,
+                TaskType.ALIGN,
+                TaskType.TRANSCRIBE,
+            ]
```

Comment:
We should probably just get rid of this and require task_types to be set (in a new PR)